service/src/test/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelperTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JudgeAndLegalAdvisorHelperTest {
+    @Test
+    public void givenEmptyJudgeAndLegalAdvisorGetLegalAdvisorNameShouldReturnEmptyString() {
```

Comment:
Again use Should and When in tests 👍

Reply:
Ok - will fix. Used PBANumberHelperTest for reference, all tests here used 'given' - but will update to ShouldWhen